src/lib/openapi/spec/event-schema.ts

```diff
@@ -92,6 +92,16 @@ export const eventSchema = {
             nullable: true,
             description: 'Any tags related to the event, if applicable.',
         },
+        label: {
+            type: 'string',
+            nullable: true,
+            description: 'The concise, human-readable name of the event.',
```

Comment:
Same comment as below - could we mark this as experimental somehow?

Reply:
WDYT of this? https://github.com/Unleash/unleash/pull/8205/commits/8432e536a72e61db763f05b96f2ba7f1c3c4f4a5

Reply:
Yeah that'll work! Should be super clear :)